pkg/skaffold/deploy/deploy_mux.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deploy
+
+import (
+	"bytes"
+	"context"
+	"io"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+)
+
+// DeployerMux forwards all method calls to the deployers it contains.
+// When encountering an error, it aborts and returns the error. Otherwise,
+// it collects the results and returns it in bulk.
+type DeployerMux []Deployer
+
+var _ Deployer = DeployerMux{}
```

Comment:
What does this line do?

Reply:
gotcha, it is just a compile time check to make sure that `DeployerMux` implements the `Deployer` interface.

Reply:
a comment would be nice?

Reply:
Exactly, this was my local compile check that I have implemented the interface. As this pattern seems to be unused in Skaffold, I opted to remove the line instead.